apps/docs/src/.vitepress/components/ComponentGrid.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import ComponentCard, { type ComponentCardProps } from "./ComponentCard.vue";
+
+export type ComponentGridProps = {
+  /** Description to show above the grid. */
+  description: string;
+  /** List of components to show. */
+  components: ComponentCardProps[];
+  /** Timestamp of estimated due date (will be formatted as month + year). */
+  dueDate?: string;
```

Comment:
```suggestion
  dueDate?: Date;
```

Why do we need to convert the date in index.data.ts to string and back to date here? is a Date type not working for props?

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/7a42c92e-5a9b-4910-aff4-1f742806b407)
Ah, I see, does not work.